samza-core/src/main/scala/org/apache/samza/diagnostics/DiagnosticsManager.java

```diff
@@ -80,20 +82,20 @@
   private final SystemStream diagnosticSystemStream;
 
   public DiagnosticsManager(String jobName, String jobId, Map<String, ContainerModel> containerModels,
-      Integer containerMemoryMb, Integer containerNumCores, Integer numStoresWithChangelog, String containerId,
-      String executionEnvContainerId, String taskClassVersion, String samzaVersion, String hostname,
+      Integer containerMemoryMb, Integer containerNumCores, Integer numStoresWithChangelog, Long maxHeapSizeBytes, Integer containerThreadPoolSize,
```

Comment:
nit: can we add the new parameters at the end?

Reply:
we could, i wanted to try and keep the sizing-related params together, container-id related together, and the remaining at the end.